### DISCUSSION TITLE
Revive the web-tasks connector from Incubator

### DISCUSSION THREAD

**Discussion by gnl42** [eclipse-mylyn/org.eclipse.mylyn#609] [General]
I have need of the web-tasks connector for work. 

Are there any plans for the Incubator project? 

Would it be worth while to add  to the mylyn project? Or fork it and take it over?

**Comment by ruspl-afed** [comment]
You can open a PR for mylyn project with things you need from mylyn-incubator. 
Since it was a part of Eclipse Mylyn some time ago I do not expect license issues with its sources.
If it needs any additional 3rd party we should discuss each of them individually,

Or/and you can fork it and do what you need according to its license.

**Comment by gnl42** [comment]
Sigh, merging turned out to be pretty easy. But when I tried it I was unable to make it work against my work system. So it's useless for my purposes. 

Worth adding to the base code?

**Comment by gnl42** [comment]
Either that or we should remove the references to the connector from the Mylyn documentation
